From e692723bf66c2334bddc1dbd4da8d695035272d6 Mon Sep 17 00:00:00 2001
From: Mian Yousaf Kaukab <yousaf.kaukab@intel.com>
Date: Thu, 6 Nov 2014 09:39:25 +0100
Subject: [PATCH] usb: composite: allow gadget driver to provide kobj reference

Function driver can use it for example to create sysfs nodes.

Change-Id: Ie6d6aa929ce0d1720875a487a5809f2421fee01c
Signed-off-by: Mian Yousaf Kaukab <yousaf.kaukab@intel.com>
---
 include/linux/usb/composite.h | 1 +
 1 file changed, 1 insertion(+)

diff --git a/include/linux/usb/composite.h b/include/linux/usb/composite.h
index e5c673b..3207313 100644
--- a/include/linux/usb/composite.h
+++ b/include/linux/usb/composite.h
@@ -463,6 +463,7 @@ struct usb_function_driver {
 	struct list_head list;
 	struct usb_function_instance *(*alloc_inst)(void);
 	struct usb_function *(*alloc_func)(struct usb_function_instance *inst);
+	struct kobject *parent;
 };
 
 struct usb_function_instance {
-- 
1.9.1

